plugin/transfer/setup.go

```diff
@@ -37,6 +37,10 @@ func setup(c *caddy.Controller) error {
 				continue
 			}
 			t.Transferers = append(t.Transferers, tr)
+			ch := tr.Notify()
+			if ch != nil {
+				go t.Notify(ch)
```

Comment:
how is this go routine terminated?

Reply:
IIUC, `t.Notify` will exit when the channel closes.  As is now, it would be up to the plugins that write to the channels to close their channel when stopping.

Reply:
I have added a note in the Transferer interface Notify method docs to this effect.